dcap_ql/sys/src/lib.rs

```diff
@@ -1,7 +1,6 @@
 // Copyright (c) 2022 The MobileCoin Foundation
 //! FFI functions for the SGX SDK DCAP ql library.
 
-#![no_std]
```

Comment:
Is there a reason to remove this if the bindings don't require `std`?

Reply:
Thinking about this more, I think you're spot on and that there isn't a good reason in this instance.
Doing some digging around it seems that the main downside of using `no_std` in a crate is that it's part of your _contract_ to users.  So if in the future you end up needing to depend on `std` for anything you could potentially break your users.  I don't see that happening for the FFI bindings.  The rust wrappers coming next should probably avoid `no_std` unless they are enclave crates, but for these FFI bindings I think `no_std` would probably be good.

@jcape do you have thoughts here?

I'm merging this as is and we can come back with another PR to standardize since currently I see 4 other sys crates without `no_std`
> ➜  sgx git:(develop) fd lib | xargs rg no_std --files-without-match  
./urts/sys/types/src/lib.rs
./urts/sys/src/lib.rs
./urts/src/lib.rs
./test_enclave/src/lib.rs
./quote_verify/src/lib.rs
./dcap_quoteverify/sys/src/lib.rs
./core/build/src/lib.rs
./capable/sys/src/lib.rs
./capable/src/lib.rs